docs/public/index.css

```diff
@@ -23,7 +23,7 @@ body {
   font-size: clamp(
     0.875rem,
     0.4626rem + 1.0309vw + var(--user-font-scale),
-    1.125rem
+    1.0rem
```

Comment:
Agree that the font was too large! This kinda solves it, but adjusting the overall `clamp` would be better. I'll push a fix here.